hpat/hiframes/hiframes_typed.py

```diff
@@ -314,6 +314,7 @@ def _run_getattr(self, assign, rhs):
             nodes.append(assign)
             return nodes
 
+        # PR171. This needs to be commented out
```

Comment:
What if comment this out of the code? I mean, if it will work with HPAT compiler pipeline it is better to comment it out.